omnigibson/envs/env_base.py

```diff
@@ -403,6 +426,10 @@ def step(self, action):
         # Grab observations
         obs = self.get_obs()
 
+        # Step the scene graph builder if necessary
+        if self._scene_graph_builder is not None:
+            self._scene_graph_builder.step(self.scene)
```

Comment:
this is cool!! does updating the scene graph cause any significant slowdowns? if so, might be good to have a initialization flag when the Env is created to determine whether the scene graph should be created?